docker-compose.yml

```diff
@@ -70,10 +51,24 @@ services:
         max-file: "5"
 
   adm:
+    profiles:
+      - all
+      - ode_full
+      - ode_base
+      - adm
     build: ./asn1_codec
     image: jpoode_acm:latest
+    restart: ${RESTART_POLICY}
```

Comment:
Previously, we were restarting on failure since the ACM exits if it can't find Kafka. With the current configuration, the ADM & AEM services are giving up after 3 failures. We should either hardcode a restart policy of "on-failure" for the adm/aem or implement some sort of health check.

Reply:
This is now addressed by using health checks on services that require kafka/ode to be running.